spyder_terminal/widgets/terminalgui.py

```diff
@@ -88,7 +88,7 @@ class TerminalWidget(QFrame, SpyderWidgetMixin):
     terminal_ready = Signal()
 
     def __init__(self, parent, port, path='~', font=None, theme=None,
-                 color_scheme=None):
+                 color_scheme=None, config_id=None):
```

Comment:
```suggestion
        color_scheme=None,
        config_id=None,
    ):
```

Use Black formatting.